docs/backlog-management.md

```diff
@@ -0,0 +1,79 @@
+# Issue Management for OpenTelemetry
+
+It's an important community goal for OpenTelemetry that our members find the backlogs to be responsive, and easy to take part in. Having a shared methodology for processing issues helps new members understand the current state of each project.
```

Comment:
> Having a shared methodology for processing issues helps new members understand the current state of each project

I don't understand this goal:

- Do you mean it will help contributors (members). Or it also will help end users?
- Do you propose some metric that new member can evaluate? Or the project state will be expressed by number of stale issues?

Maybe just replace the entire sentence with "Shared practices will simplify collaboration and engagement as well as help standardize on automation and overall project management."